src/plugins/embeddable/common/types.ts

```diff
@@ -52,6 +53,11 @@ export type EmbeddableInput = {
    */
   syncColors?: boolean;
 
+  /**
+   * Flag indicating that the chart is rendered in a non-interactive environment.
+   */
+  noInteractivity?: boolean;
```

Comment:
we don't need to add this, `disableTriggers` is the existing embeddable flag we should use.

Reply:
Thanks 👍  I've replaced that.